homeassistant/components/zha/core/channels/homeautomation.py

```diff
@@ -75,7 +75,6 @@ async def async_update(self):
 
     async def async_initialize(self, from_cache):
         """Initialize channel."""
-        await self.get_attribute_value("active_power", from_cache=from_cache)
```

Comment:
This also removes the read of `active_power` was this intentional?

Reply:
It is read in super ().async_initialize(), just removed an extra request here.